privacy.html

```diff
@@ -260,7 +260,7 @@ <h2>How should this specification work in the context of a user agent’s "incog
       <section>
         <h2>Does this specification persist data to a user’s local device?</h2>
         <p>
-The overall life cycle of Verifiable Claims envisages that they are stored persistently on the user’s local device or on a remote device under the user’s control. However, Verifiable Claims on their own, if captured by a hostile entity, should not be of any value to it, except in so far as the Verifiable Claim may potentially reveal a small amount of Personally Identifiable Information (PII). about the subject.
+The overall life cycle of Verifiable Claims envisages that they are stored persistently on the user’s local device or on a remote device under the user’s control. However, Verifiable Claims on their own, if captured by a hostile entity, should not be of any value to it, except in so far as the Verifiable Claim may potentially reveal a small amount of Personally Identifiable Information (PII) about the subject.
```

Comment:
["insofar" is US English](https://www.google.com/search?q=%22in+so+far%22+or+%22insofar%22), which is [W3C's standard](https://www.w3.org/Guide/manual-of-style/#Spelling).
```suggestion
The overall life cycle of Verifiable Credentials envisages that they are stored persistently on a user’s local device or on a remote device under the user’s control. However, Verifiable Credentials on their own, if captured by a hostile entity, should not be of any value to it, except insofar as the Verifiable Credential may potentially reveal a small amount of Personally Identifiable Information (PII) about the subject.
```

Reply:
Now in #1288.